config/eslint-config/README.md

```diff
@@ -10,42 +10,55 @@ npm install -D @ilo-org/eslint-config
 
 ## Usage
 
-You can use `@ilo-org/eslint-config` in your project by extending it in your
-`eslint` configuration. For example, if we had an `.eslintrc` file:
+`@ilo-org/eslint-config` provides a list of ESLint flat configurations:
 
-```json
-{
-  "extends": ["@ilo-org/eslint-config"]
-}
+| Config Name | Description                           | Internal Extends |
+| ----------- | ------------------------------------- | ---------------- |
+| js          | Base JavaScript config                | ⛔               |
+| recommended | Recommended configuration `js` + `ts` | `js`             |
+| react       | React Typescript config               | `ts` -> `js`     |
+
+You can use any configuration you like by simply including it inside your eslint config
+
+Just export it
+
+```js
+import configs from "@ilo-org/eslint-config";
+
+export default configs.recommended;
 ```
 
-## TypeScript
-
-To use this configuration in a project with Typescript, you can add an `overrides` property to your `.eslintrc`. Here's an example:
-
-```cjs
-// .eslintrc.cjs
-
-module.exports = {
-  extends: ["@ilo-org/eslint-config"],
-  ignorePatterns: ["node_modules/"],
-  overrides: [
-    /* =================== */
-    /* TypeScript Settings */
-    /* =================== */
-    {
-      // Which files the override will apply to relative to the package root
-      files: ["**/*.{ts,tsx}"],
-      extends: ["@ilo-org/eslint-config/typescript"],
-      // Your typescript parser options
-      parserOptions: {
-        project: ["./tsconfig.json"],
-      },
+or extend and add your own
+
+```js
+import configs from "@ilo-org/eslint-config";
+
+export default [
+  ...configs.recommended,
+  {
+    rules: {
+      "no-console": "warn",
     },
-  ],
-};
+  },
+];
 ```
 
+## Opinionated rules
+
+| Rule                                              | Status | Description                                                     |
```

Comment:
The status field of this table is a bit confusing. Does it mean that the config errors out for certain rules while warning about others?

A description about the status will be useful.

Reply:
In ESLint lingo, this is called severity - https://eslint.org/docs/latest/use/configure/rules#rule-severities